components/content/LoadingText/LoadingText.ts

```diff
@@ -0,0 +1,32 @@
+import { LitElement } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators.js';
+import { unsafeStatic, html } from 'lit/static-html.js';
+
+@customElement('diamond-loading-text')
+export class LoadingText extends LitElement {
+  @property({ reflect: true }) as?:
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'p'
+    | 'div'
+    | 'span' = 'p';
+
+  getTagName() {
+    return this.as ?? 'p';
+  }
+  render() {
+    const tag = this.getTagName() || 'p';
+    return html`<${unsafeStatic(tag)}><slot></slot></${unsafeStatic(tag)}>`;
```

Comment:
Is unsafeStatic OK to use? What are the security risks?

Reply:
It's an XSS vulnerability if you allow users to put anything they want into the prop field, but we have a set of prop options here so I think it's fine and would just trigger a typescript compilation error with an invalid value?

Reply:
Sweet